docs/pfsense_installerconfig.md

```diff
@@ -1,11 +1,87 @@
-Unattanded installation PFSense
+Unattended installation PFSense
 ===============================
 
+This document describes the process of preparing the automatic installer pFsense
+on a USB stick. The whole procedure was tested on Ubuntu with the kernel
+5.15.0-58-generic, the download and installation procedure of the kernel is also
```

Comment:
The naming convention of kernels may be different in other distros. At least mention it was done on Ubuntu

Reply:
Redundant info was removed